script/install_oneapi.sh

```diff
@@ -42,3 +57,4 @@ which "${CXX}"
 ${CXX} -v
 which "${CC}"
 ${CC} -v
+clinfo -l
```

Comment:
I'd suggest
```suggestion
sycl-ls
```
because it also include the Level-0 devices